ioctl/cmd/ws/wsprojectconfig.go

```diff
@@ -42,8 +42,12 @@ var (
 			if err != nil {
 				return errors.Wrap(err, "failed to get flag expand-param")
 			}
+			outputFile, err := cmd.Flags().GetString("output-file")
```

Comment:
will err be returned without output flag?

Reply:
keep same with other param's error